ts_src/issuance.ts

```diff
@@ -72,10 +72,7 @@ export function hashContract(contract: IssuanceContract): Buffer {
     {},
   );
 
-  return bcrypto
-    .sha256(Buffer.from(JSON.stringify(sortedContract)))
-    .slice()
-    .reverse();
+  return bcrypto.sha256(Buffer.from(JSON.stringify(sortedContract))).slice();
```

Comment:
could you remove also the `slice()` method ? we used it to reverse the bytes safely but now it's not useful anymore.